migrate-users-between-project/README.md

```diff
@@ -0,0 +1,69 @@
+### MongoDB Support Tools
+=====================
+
+# 🛠️ MongoDB Atlas User & Role Migration Tool
+
+This tool provides a Node.js-based CLI utility to **migrate database users and custom roles** from one MongoDB Atlas project to another using the [Atlas Admin API](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/).
+
+---
+
+## 📦 Prerequisites
+
+Before using this tool, ensure you have:
+
+- Node.js installed (v16 or above)
+- MongoDB Atlas Public and Private API Keys
+- Source Project ID and Destination Project ID
+- Network access to `https://cloud.mongodb.com`
+
+---
+
+## 🚀 Getting Started
+
+1. **Clone or Download the Repository**
+
+   ```bash
+   git clone https://github.com/mongodb/support-tools/migrate-users-between-project
+   cd migrate-users
+  
+2. **Install Dependencies**
+
+   ```bash
+   npm install
+
+3. **Configure Your Environment**
+
+Edit the relevant variables inside config.js:
```

Comment:
It would improve usability if we included links / more information on what these values mean.